AlphaWallet/Settings/ViewControllers/SettingsViewController2.swift

```diff
@@ -0,0 +1,140 @@
+            switch section {
+            case 0:
+                return settingsWalletData.count
+            case 1:
+                return settingsSystemData.count
+            case 2:
+                return settingsHelpData.count
+            default:
+                return settingsFooterData.count
+            }
+        }
+        
+        func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+            if indexPath.section == 3 {
+                let cells = tableView.dequeueReusableCell(withIdentifier: "appInfoCell", for: indexPath) as? AppInfoTableCell
+                              cells!.selectionStyle = .none
+                              cells!.settings = settingsFooterData[indexPath.row]
+                return cells!
+            } else {
+                let cell = tableView.dequeueReusableCell(withIdentifier: "settingCell", for: indexPath) as! SettingTableViewCell
+                cell.selectionStyle = .none
+                cell.accessoryType = .disclosureIndicator
+                cell.delegate = self
+                if indexPath.row == 1 && indexPath.section == 1 {
+                    cell.tableSwitch.isHidden = false
+                    cell.accessoryType = .none
+                }
+                switch indexPath.section {
+                case 0:
+                    cell.settings = settingsWalletData[indexPath.row]
+                case 1:
+                    cell.settings = settingsSystemData[indexPath.row]
+                case 2:
+                    cell.settings = settingsHelpData[indexPath.row]
+                default:
+                   cell.settings = settingsHelpData[indexPath.row]
+                }
+                return cell
+            }
+        }
+        
+        func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+            if indexPath.row == 1 && indexPath.section == 0 {
+                return 80
+            } else if indexPath.section == 3 {
+                return  50
+            }
+            return 60
+        }
+    
+        func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+            if section == 3 {
+                return 0
+            }
+            return 50
+        }
+
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let headerView = tableView.dequeueReusableHeaderFooterView(withIdentifier: "SettingHeaderView") as! SettingViewHeader
+        if section == 0 {
+            headerView.title = "WALLET"
+        } else if section == 1 {
+            headerView.title = "SYSTEM"
+        } else {
+            headerView.title = "HELP"
+        }
+        return headerView
+    }
+}
+extension SettingsViewController2: SettingTableViewCellDelegate {
+    func onOffPassCode(cell: SettingTableViewCell, switchState isOnOff: Bool) {
```

Comment:
`isOnOff: Bool`

You probably meant `isOn: Bool` instead.

Reply:
Done